salesforcecdpconnector/authentication_helper.py

```diff
@@ -181,3 +186,33 @@ def _token_by_un_pwd_flow(self, login_url, client_id, client_secret, username, p
             return self._exchange_token(org_url, core_token)
         else:
             raise Error('Core token retrieval failed with code %d' % access_code_res.status_code)
+        
+    def _token_by_jwt_bearer_flow(self, login_url, username, client_id, private_key):
```

Comment:
@deejay-hub : Can you please add some unit test for this?

Reply:
Added method ` test_token_by_jwt_bearer_flow` to **test_authentication_helper.py**